generic-styles/content-common.less

```diff
@@ -295,33 +295,21 @@ blockquote {
     padding: 0.5em 1em;
   }
 
-  // Exercise Solutions contain a button to toggle showing or hiding the solution
+  // Exercise Solutions contain a summary to toggle showing or hiding the solution
   [data-type="solution"],
   .solution {
     border-top: 0.1rem solid @gray;
 
     // The Show/Hide Solution text
-    > .ui-toggle-wrapper {
+    > .ui-toggle {
```

Comment:
we have to update the baked styles to match this as a separate change

https://github.com/openstax/cnx-recipes/blob/main/styles/designs/webview/parts/_exercise-components.scss#L56

Reply:
Fixed conflicts 👍 For the recipe — now that the solution is also the ui-toggle-wrapper, maybe that gets removed? I'm realizing now that it may be unnecessary to have the wrapper class if it isn't being targeted by any CSS. I'll take a look at the recipe code and get a better understanding for how it's used.

Reply:
the recipe styles are not currently used, that is something that we've been recently developing and hope to move over to in the near future, so we're in the awkward phase where style changes need to be maintained in two places. unfortunately bartek was doing most of the work on that and has now left, so we'll need some other members of the team to spin up on how the "baked styles" are set up